config/secrets.yml

```diff
@@ -1,2 +1,24 @@
 development:
-    mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
\ No newline at end of file
+    mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
```

Comment:
You can create a default environment with all variables in common and then inherit it from the others